Client/js/admin_categories.js

```diff
@@ -7,12 +7,21 @@ function getCategorisList() {
             '<td>' + sortedList[i].id + '</td>' +
             '<td>' + sortedList[i].name + '</td>' +
             '<td>' + sortedList[i].imageUrl + '</td>' +
+            '<td>' + '<a href="#" class="btn btn-danger delete-category">Delete</a>' + '</td>' +
             '</tr>'
             ).appendTo('.categories-table tbody');
+
+            var categoryId = sortedList[i].id;
+            $(".delete-category").click(function() {
```

Comment:
Here you subscribe on click of all ".delete-category" anchors, but you need to add handler to added anchor only.